.github/workflows/storybook.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and Deploy
+on:
+  push:
+    paths: ["stories/**", "src/components/**"]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout ️
+        uses: actions/checkout@v2.3.1
+        with:
+          persist-credentials: false
+      - name: Install and Build
+        run: |
+          npm install
+          npm run build-storybook
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@3.6.2
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: main
+          FOLDER: docs-build
```

Comment:
Can this be set to `storybook-static`, as that's the original Storybook folder, and is already in `.gitignore`?

Reply:
Done :)